src/RestaurantCreateContainer.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+
+import { useDispatch, useSelector } from 'react-redux';
+
+import actions from './actions';
```

Comment:
```suggestion
import {
  addRestaurant,
  changeRestaurantField,
} from './actions';
```

action은 뽀개서 가져오는 게 편해요~